src/platform/site-wide/mhv/utilities.js

```diff
@@ -0,0 +1,37 @@
+import environment from 'platform/utilities/environment';
+import { eauthEnvironmentPrefixes } from 'platform/utilities/sso/constants';
+
+const eauthPrefix = eauthEnvironmentPrefixes[environment.BUILDTYPE];
+const mhvPrefix = environment.isProduction() ? 'www' : 'mhv-syst';
+
+// TODO: Add labs-and-tests route
+const mhvToEauthRoutes = {
+  'download-my-data': 'eauth?deeplinking=download_my_data',
+  'web/myhealthevet/refill-prescriptions':
+    'eauth?deeplinking=prescription_refill',
+  'secure-messaging': 'eauth?deeplinking=secure_messaging',
+  appointments: 'eauth?deeplinking=appointments',
+  home: 'eauth',
+};
+
+// An MHV URL is a function of the following parameters:
+// 1. Whether this is a production or staging environment
+// 2. Whether SSOe is in use (enabled site wide, and for this particular user)
+// 3. The specific MHV path being accessed
+function mhvUrl(useSSOe, path) {
```

Comment:
callers seem to alter between using the `ssoe` feature flag and the `hasSessionSSO` function for determining the value of `useSSOe`, why not just one or the other?

Also wouldn't we only want to use the feature flag if the user is also authenticated?

Reply:
Yes, I was going to remove the uses of hasSessionSSO. 

I think all the uses of this method already enforce that the user needs to be logged in, but I'll see about enforcing that here.